projects/core/components/textfield/textfield.component.ts

```diff
@@ -137,6 +141,11 @@ export class TuiTextfieldComponent<T> implements TuiDataListHost<T> {
         this.open.set(false);
     }
 
+    @ContentChildren(TUI_AUXILIARY, {descendants: true})
+    protected set auxiliariesSetter(x: ReadonlyArray<Type<unknown>>) {
```

Comment:
Is it readonly array here and not query list? Are we sure it is triggered everytime it changes and we shouldn't instead listen to changes on the query list?